### PR TITLE
Update conformer recipes

### PR DESCRIPTION
**Pull request by VahidooX**

### CODE REVIEW

File: nemo/collections/asr/models/ctc_bpe_models.py [hunk 1]
```diff
@@ -77,21 +77,42 @@ def list_available_models(cls) -> Optional[PretrainedModelInfo]:
...
             pretrained_model_name="stt_en_conformer_ctc_small",
             description="For details about this model, please visit https://ngc.nvidia.com/catalog/models/nvidia:nemo:stt_en_conformer_ctc_small",
-            location="https://api.ngc.nvidia.com/v2/models/nvidia/nemo/stt_en_conformer_ctc_small/versions/1.0.0rc1/files/stt_en_conformer_ctc_small.nemo",
+            location="https://api.ngc.nvidia.com/v2/models/nvidia/nemo/stt_en_conformer_ctc_small/versions/1.0.0/files/stt_en_conformer_ctc_small.nemo",
         )
         results.append(model)
 
         model = PretrainedModelInfo(
-            pretrained_model_name="stt_en_conformer_ctc_medium",
+            pretrained_model_name="stt_en_conformer_ctc_medium_",
```
Comment (titu1994) [inline nemo/collections/asr/models/ctc_bpe_models.py:85]: Why the extra underscore ?

**Review by titu1994 — APPROVED** [review]
Looks great. I will update the Citrinet configs to match this style, more useful.

File: nemo/collections/asr/models/ctc_bpe_models.py (hunk 1 above)
Comment (VahidooX) [reply nemo/collections/asr/models/ctc_bpe_models.py]: My mistake! Fixed it.

File: tests/core/test_save_restore.py [hunk 2]
```diff
@@ -134,12 +134,13 @@ def test_EncDecCTCModelBPE(self):
...
-        # TODO: Switch to using named configs because here we don't really care about weights
-        cn = EncDecCTCModelBPE.from_pretrained(model_name="stt_en_conformer_ctc_small")
-        self.__test_restore_elsewhere(model=cn, attr_for_eq_check=set(["decoder._feat_in", "decoder._num_classes"]))
+    # disabled until new models are released on NGC
+    # @pytest.mark.with_downloads()
+    # @pytest.mark.unit
+    # def test_EncDecCTCModelBPE_v2(self):
+    #     # TODO: Switch to using named configs because here we don't really care about weights
+    #     cn = EncDecCTCModelBPE.from_pretrained(model_name="stt_en_conformer_ctc_small")
+    #     self.__test_restore_elsewhere(model=cn, attr_for_eq_check=set(["decoder._feat_in", "decoder._num_classes"]))
```
Comment (fayejf) [inline tests/core/test_save_restore.py:143]: Why comment it out?

**Review by fayejf — APPROVED** [review]
LGTM!

File: tests/core/test_save_restore.py (hunk 2 above)
Comment (VahidooX) [reply tests/core/test_save_restore.py:143]: This model is going to get updated with new version. So I updated the version number here, but it is not uploaded on NGC yet. We would enable it when NGC is updated.

Comment (fayejf) [reply tests/core/test_save_restore.py:143]: Got it!

File: examples/asr/conf/conformer/conformer_ctc_bpe.yaml [hunk 3]
```diff
@@ -1,19 +1,42 @@
 # It contains the default values for training a Conformer-CTC ASR model, large size (~120M) with CTC loss and sub-word encoding.
-# Default learning parameters in this config are set for effective batch size of 2K.
-# To train it with smaller effective batch sizes, you may need to re-tune the learning parameters or use higher accumulate_grad_batches.
 
+# Architecture and training config:
+# Default learning parameters in this config are set for effective batch size of 2K. To train it with smaller effective
+# batch sizes, you may need to re-tune the learning parameters or use higher accumulate_grad_batches.
 # Here are the recommended configs for different variants of Conformer-CTC, other parameters are the same as in this config file.
 # One extra layer (compared to original paper) is added to the medium and large variants to compensate for replacing the LSTM decoder with a linear one.
+#
```
Comment (titu1994) [inline examples/asr/conf/conformer/conformer_ctc_bpe.yaml:8]: Why are all of these things being duplicated in yaml configs ?? Put them only on NGC cards.

**Review by titu1994 — CHANGES_REQUESTED** [review]

File: examples/asr/conf/conformer/conformer_ctc_bpe.yaml (hunk 3 above)
Comment (VahidooX) [reply examples/asr/conf/conformer/conformer_ctc_bpe.yaml:8]: Which parts?
I think none of these parts are on NGC model cards.

**Review by titu1994 — APPROVED** [review]
Info about performance, LM etc belongs in their respective model cards, not in this config. Approved for now, will clean up later.

**Review by okuchaiev — CHANGES_REQUESTED** [review]
Models are not available. Can not merge it before models are published

For example:

```wget https://api.ngc.nvidia.com/v2/models/nvidia/nemo/stt_en_conformer_ctc_medium_ls/versions/1.0.0/files/stt_en_conformer_ctc_medium_ls.nemo```

**Review by titu1994 — CHANGES_REQUESTED** [review]
Since the PR needs to be cleaned anyway, please revert info about scores from the config. Place them in their respective model cards.

**Review by titu1994 — DISMISSED** [review]
Thanks for the cleanup. Lets merge once the checkpoint links work.

**Comment by okuchaiev** [comment]
how about we remove links to non-ready checkpoints and merge this?

**Comment by titu1994** [comment]
Agreed. Everything else seems ready to merge

**Comment by VahidooX** [comment]
> 
> 
> how about we remove links to non-ready checkpoints and merge this?



> 
> 
> how about we remove links to non-ready checkpoints and merge this?

Dropped the links for new models and reverted to older checkpoints.

**Review by okuchaiev — APPROVED** [review]
lgtm, thanks!

**Pull request merged**